test/functional/functional_query_test.rb

```diff
@@ -81,6 +81,20 @@ def test_select_query_with_multibyte_column_names_results
     assert_equal 'ニ', r.row_description[1].name
   end
 
+  # Query with multiple statements is not officailly supported
+  def test_query_with_multiple_statements
+    # Results of only the last statement are obtained
+    r = @connection.query("SET SEARCH_PATH TO default; SELECT 1 as one; SELECT 2 as two")
+    assert_equal 1, r.size
+    assert_equal 1, r.row_description.length
+    assert_equal 'two', r.row_description[0].name
+
+    # Error of the former statement is raised
+    assert_raises(Vertica::Error::MissingColumn) do
+      @connection.query("SELECT missing FROM test_ruby_vertica_table; SELECT 1 FROM missing")
+    end
+  end
```

Comment:
Final question, mostly for my curiosity: 
- Does it still run the second query if thew first one fails?
- If so, does it send data to us? Is the connection in a usable state after this?